### DISCUSSION TITLE
My personal website: foreverliketh.is

### DISCUSSION THREAD

**Discussion by jackeyjack1** [thuliteio/doks#1005] [Showcase]
[Link](https://foreverliketh.is/)

I had always wanted my own website.. I am eternally grateful to Doks for giving me this opportunity! And to everyone here who I'm always reading, for helping me make sense of what the heck is going on / explaining the problems I run into. Or because you guys share some really cool trick / feature that I can incorporate. Promise to donate sometime soon! All the best!

**Comment by jackeyjack1** [comment]
Hello again! I'm working on updating my site to V1.0 but it's going a little slowly and I do have a number of questions that I keep forgetting to write down. Thank you for your continued work on this project, I really do love my site and am very grateful to Doks as my vehicle for this Personal Web journey I'm on.

Pls, I was wondering if I could be added to [the showcase?](https://getdoks.org/showcase/) I sincerely believe I have put a lot of effort into my Doks site and that it could (potentially) serve as an inspiration to others. 🙏 Let me know, and all the best!

**Comment by h-enk** [comment]
Thanks! And, thank you for using Doks 🙂 There you go:
https://getdoks.org/showcase/

Best, Henk

**Comment by jackeyjack1** [comment]
I think my site was lost in the 1.0 update and didn't make it to the new showcase... Could you add me again? 🙏 @h-enk

**Reply by h-enk** [reply]
You're right — here you are: https://getdoks.org/showcase/
